web/src/main/java/org/perfrepo/web/alerting/ConditionCheckerImpl.java

```diff
@@ -485,12 +486,12 @@ private Double getValueFromMetric(TestExecution testExecution) {
     private CommonTree parseTree(String string) {
         //lexer splits input into tokens
         ANTLRStringStream input = new ANTLRStringStream(string);
-        TokenStream tokens = new CommonTokenStream(new AlertingDSLLexer(input));
+        TokenStream tokens = new CommonTokenStream(new org.perfrepo.web.alerting.AlertingDSLLexer(input));
```

Comment:
why do we need to to use fully qualified names?

Reply:
IntelliJ doesn't like and is adding always the qualified name

Reply:
It is not me 🤣
Every time, that I open the class or edit something in this class, Intellij is adding the full qualified name. I believe that it could be an issue with Intellij and antlr3

I prefer having the full qualified name instead of dealing with "txt editor" like Sublime to remove the full qualified name.

Reply:
I prefer you fix your Inteliij instead of adding random stuff because of broken editors:-) (AFAIK @Holmistr who meinteined the code in past couple of year used Idea as well and haven't these problems, so it's very likely configuration issue)